examples/encoding_categorical_features.ipynb

```diff
@@ -136,6 +136,49 @@
     "ca_features.to_csv('ca_features.csv', index=False)\n",
     "ca_labels.to_csv('ca_labels.csv', index=False)"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": []
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Automatic generated categorical feature encoding\n",
+    "The ```categories``` argument can automatically be generated using the ```get_definitions``` and ```generate_categories``` function to respectively download the definitions available [here](https://www2.census.gov/programs-surveys/acs/tech_docs/pums/data_dict/) and create the categories dictionary.\n",
+    "Note that this feature only works for ```survey_year```>='2017' and categories not needing the externally deffined ```PUMA``` codes. [puma codes](https://www2.census.gov/geo/docs/reference/puma/2010_PUMA_Names.txt)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from folktables import ACSIncome_categories\n",
+    "\n",
+    "data_source = ACSDataSource(survey_year='2018', horizon='1-Year', survey='person')\n",
+    "ca_data = data_source.get_data(states=[\"CA\"], download=True)\n",
+    "\n",
+    "definition_df = data_source.get_definitions(download=True)\n",
+    "categories = ACSIncome.generate_categories(definition_df=definition_df)\n",
+    "\n",
+    "ca_features, ca_labels, _ = ACSIncome.df_to_pandas(ca_data, categories=categories, dummies=True)\n",
+    "\n",
+    "ca_features.to_csv('ca_features.csv', index=False)\n",
```

Comment:
Maybe we can just use .head() instead of .to_csv() as the README already shows how to save the data via pandas. This can also be changed a few cells above in the initial example.